docs/flent.md

```diff
@@ -0,0 +1,318 @@
+# Flent
+
+[Flent](https://flent.org/) is a network performance tool
+
+## Running Flent
+
+Given that you followed instructions to deploy operator,
+you can modify [cr.yaml](../resources/crds/ripsaw_v1alpha1_flent_cr.yaml)
+
+```yaml
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: flent-benchmark
+  namespace: my-ripsaw
+spec:
+  clustername: myk8scluster
+  #elasticsearch:
+  #   server: elk.server.com
+  #   port: 9200
+  #test_user: username_to_attach_to_metadata
+  workload:
+    # cleanup: true
+    name: flent
+    args:
+      #image: quay.io/username/reponame:tagname
+      serviceip: false
+      hostnetwork: false
+      multus:
+        enabled: false
+      pin: false
+      pin_server: "node-0"
+      pin_client: "node-1"
+      pair: 1
+      test_types:
+        - tcp_download
+      runtime: 30
+```
+`test_types` is a list of all tests it should run. Not every test has been tested with benchmark-operator
```

Comment:
@jaredoconnell Can you mention what has been tested?

Reply:
I tested everything in bulk and have clarified what works and what is most tested. Is that good?